core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala

```diff
@@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
 import org.apache.spark.SecurityManager
 import org.apache.spark.metrics.MetricsSystem
 
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
```

Comment:
I think we should reorder imports. (Please see [databricks/scala-style-guide#imports](https://github.com/databricks/scala-style-guide#imports).)